From 05acb6ecc8eb7426c4664a1e8fd22ad69256d541 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Sun, 26 Jan 2020 15:36:07 +0200
Subject: [PATCH] arm64: dts: lx2160a-cex7: add on-module eeproms

This patch adds 4 eeprom support on i2c mux channel #0 -
1. Bootable 512Kbit eeprom at address 0x50.
2. Memory SO-DIMMs SPD channels at 0x51 (upper SO-DIMM) and 0x53.
3. 2Kb eeprom at 0x57 will be used by SolidRun to hold manufacturing
data.

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 .../boot/dts/freescale/fsl-lx2160a-cex7.dts   | 22 ++++++++++++++++++-
 1 file changed, 21 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-lx2160a-cex7.dts b/arch/arm64/boot/dts/freescale/fsl-lx2160a-cex7.dts
index 1c1a0d47897d..2b8f1118b37a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-cex7.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-cex7.dts
@@ -81,7 +81,27 @@ I2C switch -
 		reg = <0x77>;
 		#address-cells = <1>;
 		#size-cells = <0>;
-
+		i2c@0 {
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x0>;
+			24aa512@50 {
+				compatible = "atmel,24c512";
+				reg = <0x50>;
+			};
+			spd1@51 {
+				compatible = "atmel,spd";
+				reg = <0x51>;
+			};
+			spd2@53 {
+				compatible = "atmel,spd";
+				reg = <0x53>;
+			};
+			m24c02@57 {
+				compatible = "atmel,24c02";
+				reg = <0x57>;
+			};
+		};
 		i2c@1 {
 			#address-cells = <1>;
 			#size-cells = <0>;
-- 
2.17.1

